webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/git/ZipUtils.scala

```diff
@@ -193,7 +200,12 @@ object ZipUtils {
     }
     val base = file.getParentFile.toURI
 
-    def getPath(f: File) = base.relativize(f.toURI).getPath
+    def getPath(f: File): Option[String] = {
+      base.relativize(f.toURI).getPath match {
+        case null => None
+        case p    => Some(p)
+      }
```

Comment:
I specifically avoided that one because it is not clear at all and a java thing. It's really hard to read that bit and infer in what case you get what. 
(one of scala API design failt - `Option.fromNullable` would have been so much better)